src/databaseTests/src/test/kotlin/org/vaccineimpact/api/databaseTests/tests/UserTests.kt

```diff
@@ -597,7 +599,7 @@ class UserTests : RepositoryTests<UserRepository>()
             val hash = it.dsl.select(APP_USER.PASSWORD_HASH)
                     .from(APP_USER)
                     .where(APP_USER.USERNAME.eq("will"))
-                    .fetchOne().value1()
+                    .fetchSingle().value1()
```

Comment:
`fetchOne` is nullable but I guess wasn't marked as such before, so everywhere we expect a non-null value I've changed to `fetchSingle`